.github/workflows/build-client.yaml

```diff
@@ -62,6 +62,6 @@ jobs:
           file: cmd/kubectl-k8ssandra/Dockerfile
           push: ${{ github.event_name != 'pull_request' }}
           tags: k8ssandra/k8ssandra-client:${{ steps.vars.outputs.sha_short }}, k8ssandra/k8ssandra-client:latest
-          platforms: linux/amd64
+          platforms: linux/amd64,linux/arm64
```

Comment:
question: Did you check that this works? We ran into the following issue with multi platform builds:

```
ERROR: docker exporter does not currently support exporting manifest lists
Error: buildx failed with: ERROR: docker exporter does not currently support exporting manifest lists
```

Reply:
That error happens if "load" is set to true. We don't have load here as the image itself is not used (it's only pushed).